timestamp-impl/src/main/java/com/palantir/timestamp/TimestampAllocationFailures.java

```diff
@@ -32,6 +32,7 @@
     private final Logger log;
 
     private volatile Throwable previousAllocationFailure;
```

Comment:
But as discussed let's do this separately, and if/when we do this we should really document why it's safe